crates/pep508-rs/src/lib.rs

```diff
@@ -463,14 +463,25 @@ pub enum VersionOrUrl {
 }
 
 /// Unowned version specifier or URL to install.
-#[derive(Debug, Clone, Eq, Hash, PartialEq)]
+#[derive(Debug, Clone, Copy, Eq, Hash, PartialEq)]
```

Comment:
This pacifies clippy's needless pass by value

Reply:
Yeah fine for this to be copy.